wordle-web/src/views/GameView.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="game">
+        <v-card>
+          <v-card-title>
+              <h1>Word War 1</h1>
+          </v-card-title>
+
+          <v-card-subtitle>
+              <p>Title work in progress</p>
```

Comment:
Same here, your v-card-subtitle tag is all you need
```suggestion
              Title work in progress
```